content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+    {{% /note %}}
+
+    Run the following command to give InfluxDB read and write permissions on the certificate files.
+
+    ```bash
+    sudo chmod 644 /etc/ssl/<CA-certificate-file>
+    sudo chmod 600 /etc/ssl/<private-key-file>
+    ```
+
+3. **Run `influxd` with TLS flags**
+
+    Start InfluxDB with TLS command line flags:
+
+    ```bash
+    influxd \
+    --tls-cert "/etc/ssl/influxdb-selfsigned.crt"\
+    --tls-key "/etc/ssl/influxdb-selfsigned.key"
+    ```
+
+That's it! You've successfully set up HTTPS with InfluxDB.
+
+## Set up HTTPS with a self-signed certificate
+
+1. **Generate a self-signed certificate**
+
+    The following command generates a private key file (`.key`) and a self-signed
+    certificate file (`.crt`) which remain valid for the specified `NUMBER_OF_DAYS`.
+    It outputs those files to `/etc/ssl/` and gives them the required permissions.
+
+    ```bash
+    sudo openssl req -x509 -nodes -newkey rsa:2048 \
+    -keyout /etc/ssl/influxdb-selfsigned.key \
+    -out /etc/ssl/influxdb-selfsigned.crt \
+    -days <NUMBER_OF_DAYS>
+    ```
+
+    When you execute the command, it will prompt you for more information.
+    You can choose to fill out that information or leave it blank; both actions generate valid certificate files.
+
+2. **Run `influxd` with TLS flags**
+
+    Start InfluxDB with TLS command line flags:
+
+    ```bash
+    influxd \
+    --tls-cert "/etc/ssl/influxdb-selfsigned.crt" \
+    --tls-key "/etc/ssl/influxdb-selfsigned.key"
+    ```
+
+That's it! You've successfully set up HTTPS with InfluxDB.
+
+## Connect Telegraf to a secured InfluxDB instance
+
+Connecting [Telegraf](/telegraf/latest/) to an InfluxDB instance that's using
+HTTPS requires some additional steps.
+
+In the Telegraf configuration file (`/etc/telegraf/telegraf.conf`), edit the `urls`
+setting to indicate `https` instead of `http`.
+(Change `localhost` to the relevant domain name if necessary.)
+If you're using a self-signed certificate, uncomment the `insecure_skip_verify` setting and set it to `true`.
+
+```toml
+    ###############################################################################
+    #                            OUTPUT PLUGINS                                   #
+    ###############################################################################
+>
+    # Configuration for InfluxDB server to send metrics to
+    [[outputs.influxdb]]
+      ## The full HTTP or UDP endpoint URL for your InfluxDB instance.
+      ## Multiple urls can be specified as part of the same cluster,
+      ## this means that only ONE of the urls will be written to each interval.
+      # urls = ["udp://localhost:8089"] # UDP endpoint example
+      urls = ["https://<domain_name>.com:8086"]
+>
+    [...]
+>
+      ## Optional SSL Config
+      [...]
+      insecure_skip_verify = true # <-- Update only if you're using a self-signed certificate
+```
+
+Next, restart Telegraf and you're all set!
```

Comment:
~Next, restart Telegraf and you're all set!~ -> Restart Telegraf using the updated configuration file.